tutorial/source/tensor_shapes.ipynb

```diff
@@ -528,22 +524,22 @@
     "\n",
     "# Test with enumeration.\n",
     "enumerated = True\n",
-    "test_model(model4, guide4, TraceEnum_ELBO(max_iarange_nesting=2))"
+    "test_model(model4, guide4, TraceEnum_ELBO(max_plate_nesting=2))"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Automatic broadcasting via broadcast poutine<a class=\"anchor\" id=\"Automatic-broadcasting-via-broadcast-poutine\"></a>\n",
+    "### Automatic broadcasting inside pyro.plate<a class=\"anchor\" id=\"Automatic-broadcasting-inside-pyro-plate\"></a>\n",
     "\n",
-    "Note that in all our model/guide specifications, we hve relied on [poutine.broadcast](http://docs.pyro.ai/en/latest/poutine.html#pyro.poutine.broadcast) to automatically expand sample shapes to satisfy the constraints on batch shape enforced by `pyro.iarange` statements. However `poutine.broadcast` is equivalent to hand-annotated `.expand()` statements.\n",
+    "Note that in all our model/guide specifications, we have relied on [pyro.plate](http://docs.pyro.ai/en/latest/primitives.html#pyro.plate) to automatically expand sample shapes to satisfy the constraints on batch shape enforced by `pyro.sample` statements. However this broadcasting is equivalent to hand-annotated `.expand()` statements.\n",
     "\n",
     "We will demonstrate this using `model4` from the [previous section](#Writing-parallelizable-code). Note the following changes to the code from earlier:\n",
     "\n",
     " - For the purpose of this example, we will only consider \"parallel\" enumeration, but broadcasting should work as expected without enumeration or with \"sequential\" enumeration.\n",
-    " - We have separated out the sampling function which returns the tensors corresponding to the active pixels. Modularizing the model code into components is a common practice, and helps with maintainability of large models. The first sampling function is identical to what we had in `model4`, and the remaining sampling functions use `poutine.broadcast` to implicitly expand sample sites to confirm to the shape requirements imposed by the `iarange` contexts in which they are embedded.\n",
-    " - We would also like to use the `pyro.iarange` construct to parallelize the ELBO estimator over [num_particles](http://docs.pyro.ai/en/latest/inference_algos.html#pyro.infer.elbo.ELBO). This is done by wrapping the contents of model/guide inside an outermost `pyro.iarange` context."
+    " - We have separated out the sampling function which returns the tensors corresponding to the active pixels. Modularizing the model code into components is a common practice, and helps with maintainability of large models.\n",
+    " - We would also like to use the `pyro.plate` construct to parallelize the ELBO estimator over [num_particles]s(http://docs.pyro.ai/en/latest/inference_algos.html#pyro.infer.elbo.ELBO). This is done by wrapping the contents of model/guide inside an outermost `pyro.plate` context."
```

Comment:
Additional s `[num_particles]s(..)` is breaking the link.

Reply:
yeah plus the var name is actually `num_particles` not `num_particle`. also `latest` is no longer a branch since it is the same as `dev` so if you'll need to change all `http://docs.pyro.ai/en/latest/...` to `http://docs.pyro.ai/en/dev/...`